domains/measurements/Measurements.API/Tests/MeteringPoints/Api/v2024_01_10/MeteringPointControllerTests.cs

```diff
@@ -0,0 +1,201 @@
+
+        var response = await client.GetFromJsonAsync<GetMeteringPointsResponse>("api/meteringpoints");
+
+        response.Should().NotBeNull();
+        var settings = new VerifySettings();
+        settings.DontScrubGuids();
+        await Verifier.Verify(response, settings);
+    }
+
+    [Theory]
+    [InlineData("streetName", "", "", "")]
+    [InlineData("streetName", "buildingNo", "", "")]
+    [InlineData("streetName", "buildingNo", "floor", "")]
+    [InlineData("streetName", "buildingNo", "", "room")]
+    [InlineData("streetName", "", "floor", "room")]
+    [InlineData("streetName", "", "floor", "")]
+    [InlineData("streetName", "", "", "room")]
+    [InlineData("", "buildingNo", "floor", "room")]
+    [InlineData("", "buildingNo", "", "room")]
+    [InlineData("", "buildingNo", "floor", "")]
+    [InlineData("", "", "floor", "room")]
+    [InlineData("", "", "floor", "")]
+    [InlineData("", "", "", "room")]
+    [InlineData("", "", "", "")]
+    [InlineData("streetName", "buildingNo", "floor", "room")]
+    [InlineData("streetName", "    buildingNo", "floor", "room")]
+    [InlineData("streetName", "buildingNo", "floor", "    room")]
+    [InlineData("    streetName", "buildingNo", "floor", "room")]
+    [InlineData("    streetName", "    buildingNo", "    floor", "    room")]
+    [InlineData("streetName", "buildingNo", "    floor", "room")]
+
+    public async void EmptyAddressInformation_GetMeteringPoint_AddressLinesWithoutWhiteSpace(string streetName, string buildingNumber, string floor, string room)
+    {
+        var clientMock = Substitute.For<Meteringpoint.V1.Meteringpoint.MeteringpointClient>();
+
+        clientMock.GetOwnedMeteringPointsAsync(Arg.Any<Meteringpoint.V1.OwnedMeteringPointsRequest>())
+            .Returns(new Meteringpoint.V1.MeteringPointsResponse
+            {
+                MeteringPoints =
+                {
+                    new Meteringpoint.V1.MeteringPoint
+                    {
+                        MeteringPointId = "1234567890123456",
+                        TypeOfMp = "E17",
+                        SubtypeOfMp = "D01",
+                        StreetName = streetName,
+                        BuildingNumber = buildingNumber,
+                        FloorId = floor,
+                        RoomId = room,
+                        CityName = "City",
+                        Postcode = "1234",
+                        AssetType = "E17"
+                    }
+                }
+            });
+        _serverFixture.ConfigureTestServices += services =>
+        {
+            var mpClient = services.Single(d => d.ServiceType == typeof(Meteringpoint.V1.Meteringpoint.MeteringpointClient));
+            services.Remove(mpClient);
+            services.AddSingleton(clientMock);
+        };
+
+        var client = _serverFixture.CreateAuthenticatedClient(Guid.NewGuid().ToString());
+
+        var response = await client.GetFromJsonAsync<GetMeteringPointsResponse>("api/meteringpoints");
```

Comment:
Cant we use verifier?

Reply:
We could but this is only testing the trimming